assets/locales/nl.json

```diff
@@ -512,18 +512,21 @@
   },
   "settings": {
     "title": "Instellingen",
-    "start_qr": "QR-scanner automatisch openen na opstarten",
-    "change_pin": "Je pincode veranderen",
-    "advanced": {
-      "delete": "Verwijder alles en begin opnieuw",
-      "delete_title": "Weet je het zeker?",
-      "delete_content": "Als je doorgaat, verwijder je al je gegevens en je toegang tot de Yivi-app. Wil je de Yivi-app opnieuw gebruiken? Kies een nieuwe pincode en haal je gegevens opnieuw op.",
-      "report_errors": "Fouten naar Yivi sturen",
-      "delete_confirm": "Verwijder alles",
-      "delete_deny": "Annuleer",
-      "developer_mode": "Developer modus",
-      "enable_screenshots": "Sta screenshots toe",
-      "enable_screenshots_note": "Als dit is ingeschakeld, wordt de app niet vervaagd in de app-switcher."
+    "start_qr": "QR-scanner direct openen",
+    "start_qr_explanation": "QR-scanner automatisch openen na het opstarten van de app.",
+    "report_errors": "Fouten naar Yivi sturen",
+    "report_errors_explanation": "Door de systeem fouten naar ons op te sturen kunnen wij de gebruikerservaring verbeteren.",
```

Comment:
When I was reading the English and then the Dutch translation of `report_errors_explanation` I noticed a difference in meaning. Wouldn't it be nicer and cleaner to write the Dutch text like this? 

`Door foutrapportage in te schakelen, help je ons de gebruikerservaring te verbeteren.`